### PR TITLE
More run_microbenchmark.py cleanup

### PR DESCRIPTION
**Pull request by jtattermusch**
- run_microbenchmark.py would really need some work, but for now I'm adding a few TODOs about things I noticed.
- print summaries to the console as well (currently the jobs output is empty and the HTML report it generates is quite hard to find - since unlike on Jenkins it's only available in job's artifacts.

### CODE REVIEW

File: tools/run_tests/run_microbenchmark.py [hunk 1]
```diff
@@ -203,11 +203,27 @@ def run_summary(bm_name, cfg, base_json_name):
...
+    # both to HTML report and to console.
+    counter_heading = 'Summary: %s [with counters]' % bm_name
+    counters_summary = run_summary(bm_name, 'counters', bm_name)
+    heading(counter_heading)
+    text(counters_summary)
+    print(counters_heading)
+    print(counters_summary)
+
     if args.bq_result_table:
+        # TODO(jtattermusch): currently there is no way to differentiate
```
Comment (jtattermusch) [inline tools/run_tests/run_microbenchmark.py:225]: @ctiller  this actually seems problematic - AFAICT  there is no way of distinguishing the data from "counters" and "nocounters" runs in BQ.
Would it make sense to stop uploading the data for "counters" to at least know what's stored in BQ?

**Comment by jtattermusch** [comment]
adhoc run of performance_profile_master: http://sponge/04e728d6-1275-40c8-9ffb-29b434a5cc6c

File: tools/run_tests/run_microbenchmark.py (hunk 1 above)
Comment (ctiller) [reply tools/run_tests/run_microbenchmark.py:225]: So we smash them together right now... this code calls bm2bq.py, which ultimately calls:
https://github.com/grpc/grpc/blob/master/tools/profiling/microbenchmarks/bm_json.py#L181

and that code smashes together the times from the non-counters run with the counters from the counters run
and that combination is... probably what we actually want

**Review by ctiller — APPROVED** [review]

File: tools/run_tests/run_microbenchmark.py (hunk 1 above)
Comment (jtattermusch) [reply tools/run_tests/run_microbenchmark.py]: ok, that seems quite cryptic, so I added a TODO to add some explanatory comments in the code.

**Pull request merged**